### PR TITLE
Add longphase/phase

### PR DESCRIPTION
**Pull request by fellen31**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

File: modules/nf-core/longphase/phase/meta.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false ]`
+  - versions:
+      type: file
+      description: File containing software versions
+      pattern: "versions.yml"
+  - vcf:
```
Comment (SPPearce) [inline modules/nf-core/longphase/phase/meta.yml:73]: Should the vcf be compressed afterwards?

Comment (fellen31) [reply modules/nf-core/longphase/phase/meta.yml:73]: Should I add a bgzip within the module?

Comment (SPPearce) [reply modules/nf-core/longphase/phase/meta.yml:73]: It depends on size I guess; how big is the vcf file going to be? As well as whether the downstream tools are going to be able to read a compressed file.
The guidelines state to compress where relevant.

**Comment by fellen31** [comment]
Fixed VCF compression @SPPearce

**Review by SPPearce — APPROVED** [review]

**Pull request merged**